src/chainstate/stacks/boot/sns.clar

```diff
@@ -0,0 +1,970 @@
+;;;; Errors
+(define-constant err-panic 0)
```

Comment:
this should be consistent with the other contracts (pox, lockup) that use all caps, and underscore as the separator for errors (e.g. `ERR_STACKING_UNREACHABLE`

Reply:
Yes -- that will also help the docs generation for the boot contract output error constants.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/117b6a9965130b01d7bc216abe5da079e241ca93